pkg/operator/deploy/deploy.go

```diff
@@ -219,14 +219,20 @@ func (o *operator) resources() ([]kruntime.Object, error) {
 		Spec: arov1alpha1.ClusterSpec{
 			ResourceID:             o.oc.ID,
 			ClusterResourceGroupID: o.oc.Properties.ClusterProfile.ResourceGroupID,
-			Domain:                 domain,
-			ACRDomain:              o.env.ACRDomain(),
-			AZEnvironment:          o.env.Environment().Name,
-			Location:               o.env.Location(),
-			InfraID:                o.oc.Properties.InfraID,
-			ArchitectureVersion:    int(o.oc.Properties.ArchitectureVersion),
-			VnetID:                 vnetID,
-			StorageSuffix:          o.oc.Properties.StorageSuffix,
+			DnsmasqConfig: arov1alpha1.DnsmasqConfigSpec{
+				CacheSize:   0,
```

Comment:
Are these just being set as 0 as a default value?

Reply:
+1 here, I feel like these should be interpreted from some kind of higher-level config, that we can modify without code changes, right? (envvars, RP-Config, etc)